carma_record/scripts/carma_record.sh

```diff
@@ -4,10 +4,11 @@ while true
     do
         # Get the regex of the excluded topics
         excluded_topics="$(rosparam get exclude_regex)"
```

Comment:
If excluded_topics is unset I would think excluded topics would store something like
```
ERROR: Parameter [/excluded_topics] is not set
```
That will resolve as not empty in the following check. 
Seems like you might want an additional check for "ERROR: Parameter"

Reply:
From my testing if a param is not set then it will store an empty string, so I don't think an error check is necessary

Reply:
Seems you are correct. Looks like they must be routing the log to stderr instead of stdout so it doesn't get stored.
![image](https://user-images.githubusercontent.com/10292079/104468627-a9a36080-5585-11eb-822d-fb779d7ec196.png)